oaknational/Oak-Web-Application
# LESQ-246 fix(listing card): fix padding

## Description
padding

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-1992--oak-web-application.netlify.thenational.academy/teachers/programmes/english-secondary-ks4-edexcel/units
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
